giving-example/src/main/resources/static/adyenImplementation.js

```diff
@@ -0,0 +1,111 @@
+const clientKey = document.getElementById("clientKey").innerHTML;
+const type = document.getElementById("type").innerHTML;
+
+async function initCheckout() {
+  try {
+    const paymentMethodsResponse = await callServer("/api/getPaymentMethods");
+    const configuration = {
+      paymentMethodsResponse: paymentMethodsResponse,
+      clientKey,
+      locale: "en_US",
+      environment: "test",
+      showPayButton: true,
+      paymentMethodsConfiguration: {
+        ideal: {
+          showImage: true,
+        },
+        card: {
+          hasHolderName: true,
+          holderNameRequired: true,
+          name: "Credit or debit card",
+          amount: {
+            value: 1000,
+            currency: "EUR",
+          },
+        },
+        paypal: {
+          amount: {
+            value: 1000,
+            currency: "USD",
+          },
+          environment: "test", // Change this to "live" when you're ready to accept live PayPal payments
+          countryCode: "US", // Only needed for test. This will be automatically retrieved when you are in production.
+          onCancel: (data, component) => {
+            component.setStatus('ready');
+          },
+        }
+      },
+      onSubmit: (state, component) => {
+        if (state.isValid) {
+          handleSubmission(state, component, "/api/initiatePayment");
+        }
+      },
+      onAdditionalDetails: (state, component) => {
+        handleSubmission(state, component, "/api/submitAdditionalDetails");
+      },
+    };
+    // `spring.jackson.default-property-inclusion=non_null` needs to set in
```

Comment:
I find this comment a bit confusing, I would remove it and add a comment in `application.properties` to describe what the property is about

Reply:
I don't even know if this is still relevant to be honest, it comes from the other sample